Resources/Private/Language/de.locallang.xlf

```diff
@@ -1503,6 +1538,10 @@
 				<target>Weiter</target>
 			</trans-unit>
 
+			<trans-unit id="be.label.notification.back">
+				<source>Back to list</source>
```

Comment:
You've changed the default label from _Back_ to _Back to list_. Is this label really only used in notifications after lists and not in some other views?